todo_app/app.py

```diff
@@ -1,11 +1,27 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, url_for, request
+
+from todo_app.data import session_items
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.route("/")
 def index():
-    return 'Hello World!'
+    items = session_items.get_items()
+    return render_template("index.html", items=items)
+
+
+@app.route("/item", methods=["POST"])
+def add_item():
+    title = request.form.get("title")
+    session_items.add_item(title)
+    return redirect(url_for("index"))
+
+@app.route("/item/<id>/delete", methods=["POST"])
+def delete_item(id):
+    print(id)
```

Comment:
Do we want to keep prints on `main`? Generally, we should not have print statements in production code.

Reply:
Oops 😅